survos/crawler-bundle/1.5/config/packages/survos_crawler.yaml

```
@@ -0,0 +1,9 @@
+survos_crawler:
+# if you've run symfony proxy:domain:attach my-project, then the error messages will be displayed with this prefix.  That way, you can simply click on a link to open the page.
+#  base_url: 'https://my-project.wip/'
+    user_class: App\Entity\User
+    users:
+       - user@user.com
```

Indentation is not quite right here

---

OK, I tried it again, so that if you remove the # the spacing is correct.

---

No - the problem is with this line precisely:

```suggestion
        - user@user.com
```

---

Thanks!